packages/edit-navigation/README.md

```diff
@@ -61,9 +61,14 @@ This unlocks significant additional capabilities in the Navigation Editor. For e
 
 #### Technical Implementation details
 
-By default, `core/navigation-link` ("Navigation link") items are serialized and persisted as `nav_menu_item` posts. No serialized block HTML is stored for these standard link blocks.
+There are two types of item that can be added to a navigation:
 
-_Non_-link navigation items however, are [persisted as `nav_menu_items` with a special `type` of `block`](https://github.com/WordPress/gutenberg/blob/7fcd57c9a62c232899e287f6d96416477d810d5e/packages/edit-navigation/src/store/utils.js#L159-L166). These items have an [_additional_ `content` field which is used to store the serialized block markup](https://github.com/WordPress/gutenberg/blob/7fcd57c9a62c232899e287f6d96416477d810d5e/lib/navigation.php#L71-L101).
+1. Navigation links - these utilise the `core/navigation-link` block and are _not_ persisted as blocks.
+2. Block-based links - these are any item that is _not_ a `core/navigation-link` block. They are persisted as blocks (via a wrapper - see below).
```

Comment:
Using the terminology 'links' here might be confusing as they're often not links (spacer, search etc.)

Reply:
I'll look into this.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/34682/commits/d6bdb6ef25306e689c6313123964b052b2a25e5b